### PR TITLE
Clean rows for token-table

### PR DESCRIPTION
**Pull request by shawnxie999**
### Context of Change
Cleaning up duplicate code for .token-table rows

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

### CODE REVIEW

File: src/containers/shared/components/TokenTableRow.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const TokenTableRow = (props: TokenTableRowProps) => {
+  const { shouldDisplay = true, label, value } = props
+  return (
+    <>
+      {shouldDisplay && (
+        <tr className="row">
+          <td className="col1">{label}</td>
+          <td className="col2">{value}</td>
+        </tr>
```
Comment (mvadari) [inline src/containers/shared/components/TokenTableRow.tsx:17]: I think this is the same as `SimpleRow` for transactions. Perhaps those should be merged into something that's just `Row` or `TableRow` or something.

Comment (shawnxie999) [reply src/containers/shared/components/TokenTableRow.tsx:17]: Not sure, the token-table  is using `<table>` tag, I assume  refactoring the entire token table would be need if we want to use the SimpleRow

Comment (mvadari) [reply src/containers/shared/components/TokenTableRow.tsx:17]: Oh, good point - nvm

File: src/containers/shared/components/TokenTableRow.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  shouldDisplay?: boolean
+  label: string
+  value: any
+}
+
+export const TokenTableRow = (props: TokenTableRowProps) => {
+  const { shouldDisplay = true, label, value } = props
+  return (
+    <>
+      {shouldDisplay && (
```
Comment (mvadari) [inline src/containers/shared/components/TokenTableRow.tsx:13]: Why not just make this
```suggestion
      {value && (
```

Comment (shawnxie999) [reply src/containers/shared/components/TokenTableRow.tsx:13]: Yeah good point, I thought some of these conditions were independent of the value but there wasnt. Will change it

Comment (shawnxie999) [reply src/containers/shared/components/TokenTableRow.tsx:13]: oh the `is_burned` field is a boolean, but I cant print that in, so I had to pass in a  string

**Review by ckniffen — APPROVED** [review]

**Review by mvadari — APPROVED** [review]

**Pull request merged**